### PR TITLE
orcid auth plugin

### PR DESCRIPTION
**Pull request by c-ding-math**

### CODE REVIEW

File: src/Yesod/Auth/OAuth2/ORCID.hs [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Yesod.Auth.OAuth2.Orcid
```
Comment (pbrisbin) [inline src/Yesod/Auth/OAuth2/ORCID.hs:3]: I see the module name is `Orcid` but the file is `ORCID.hs`. I think these should match. I worry about problems on MacOS (case-insensitive file system). I would probably use `ORCID` consistently.

**Review by pbrisbin — COMMENTED** [review]
This is looking good, can you apply the Restyled fixes? There are instructions in the build log if you click the details link on the failing PR status.

File: src/Yesod/Auth/OAuth2/ORCID.hs (hunk 1 above)
Comment (c-ding-math) [reply src/Yesod/Auth/OAuth2/ORCID.hs]: I've corrected the module name.

File: src/Yesod/Auth/OAuth2/ORCID.hs [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Yesod.Auth.OAuth2.ORCID
+  ( oauth2Orcid
```
Comment (pbrisbin) [inline src/Yesod/Auth/OAuth2/ORCID.hs:4]: Thanks for correcting the module name. I think I'd like the capitalization to be consistent everywhere.

```suggestion
  ( oauth2ORCID
```

**Review by pbrisbin — CHANGES_REQUESTED** [review]

File: src/Yesod/Auth/OAuth2/ORCID.hs (hunk 2 above)
Comment (c-ding-math) [reply src/Yesod/Auth/OAuth2/ORCID.hs]: Done

**Comment by pbrisbin** [comment]
Re-opened in #186, will go out with v0.7.3.0.

**Pull request closed without merging**